tests/builders/tests/unit/filter-empty-test.ts

```diff
@@ -0,0 +1,28 @@
+import { module, test } from 'qunit';
+
+import { filterEmpty } from '@ember-data/request-utils';
+
+module('filterEmpty', function () {
```

Comment:
this is probably more useful filtering only `undefined` `null` and `[]`. `false` and `0` tend to have real meaning.
`[[]]` and similar is interesting. I think we leave those, not sure how those even serialize today would have to check